toolchain/check/testdata/class/no_prelude/name_poisoning.carbon

```diff
@@ -2,14 +2,244 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// EXTRA-ARGS: --no-dump-sem-ir
-//
```

Comment:
Why remove this?

Reply:
The sem-ir dump shows what symbols are poisoned so it's useful to verify the behavior or reason about it.

Reply:
I agree.
I do find them valuable sometimes for tests we want to pass successfully (not `fail_`) and we they can also reveal bugs when looking at PR diffs in case the test happen to not cover that without them.

Reply:
Followup: #5053.